### PR TITLE
Add qmic compilation to WiFI manual

### PR DESCRIPTION
**Pull request by RussianNeuroMancer**
Without qmic rmtfs compilation will fail, let's add it to WiFi manual

### CODE REVIEW

File: misc/lenovo-yoga-c630/wifi/README.md [hunk 1]
```diff
@@ -137,4 +139,4 @@ $  sudo systemctrl enable qrtr-ns
 $  sudo systemctrl enable pd-mapper
 $  sudo systemctrl enable tqftpserv
 $  sudo systemctrl enable rmtfs
-```
\ No newline at end of file
+```
```
Comment (lag-linaro) [inline misc/lenovo-yoga-c630/wifi/README.md]: ?

**Comment by lag-linaro** [comment]
Same questions as in #28

**Comment by RussianNeuroMancer** [comment]
Answered in #28

**Comment by lag-linaro** [comment]
Please resubmit the new links on their own.

**Pull request closed without merging**